Mixpanel/MPNotification.m

```diff
@@ -1,9 +1,51 @@
 #import "MPLogger.h"
 #import "MPNotification.h"
 #import "Mixpanel.h"
+#import "MPNetworkPrivate.h"
 
+NSString* const PROPERTY_EVAL_FUNC_NAME = @"evaluateFilters";
+NSString* const PROPERTY_FILTERS_JS_URL = @"http://cdn-dev.mxpnl.com/libs/mixpanel.property-filters.dev.js";
 @implementation MPNotification
 
+#if !TARGET_OS_WATCH
++ (JSValue*) propertyFilterFunc {
+    static JSValue* propertyFilterFunc = nil;
+    static JSContext* jsContext = nil;
+    
+    if (propertyFilterFunc != nil) {
```

Comment:
It's not thread safe for singleton. `[MPNetwork sharedURLSession]` has a good example for singleton.